user_guide_src/source/installation/upgrade_415.rst

```diff
@@ -53,6 +53,10 @@ If you want the same behavior as the previous version, set the CSRF filter like
 
 Protecting **GET** method needs only when you use ``form_open()`` auto-generation of CSRF field.
 
+.. Warning:: In general, if you use ``$methods`` filters, you should :ref:`disable auto-routing <use-defined-routes-only>`.
+    Because auto-routing permits any HTTP method to access a controller.
```

Comment:
```suggestion
.. Warning:: In general, if you use ``$methods`` filters, you should :ref:`disable auto-routing <use-defined-routes-only>`
    because auto-routing permits any HTTP method to access a controller.
```